Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla42620.cs

```diff
@@ -0,0 +1,339 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading;
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Bugzilla, 42620, "Issue Description", PlatformAffected.Default)]
+	public class Bugzilla42620 : TestContentPage // or TestMasterDetailPage, etc ...
+	{
+		public static IEnumerator<Color> Colors = ColorGenerator().GetEnumerator();
+		public static Color GetColor()
+		{
+			Colors.MoveNext();
+			var color = Colors.Current;
+			return color;
+		}
+
+		public static Tuple<int, int, int> Mix = new Tuple<int, int, int>(255, 255, 100);
+		public static IEnumerable<Color> ColorGenerator()
+		{
+			while (true)
+			{
+				var rand = new Random();
+				var r = rand.Next(256);
+				var g = rand.Next(256);
+				var b = rand.Next(256);
+
+				if (Mix != null)
+				{
+					r = (r + Mix.Item1) / 2;
+					g = (g + Mix.Item2) / 2;
+					b = (b + Mix.Item3) / 2;
+				}
+
+				yield return Color.FromRgb(r, g, b);
+			}
+		}
+
+		public View CreateBox()
+		{
+			var layout = new AbsoluteLayout();
+
+			var color = GetColor();
+
+			var box = new BoxView()
+			{
+				BackgroundColor = color
+			};
+			layout.Children.Add(box, Rectangle.FromLTRB(0, 0, 1, 1), AbsoluteLayoutFlags.All);
+
+			var label = new Label()
+			{
+				AutomationId = $"{s_id++}",
+				TextColor = Color.Black,
+				BackgroundColor = color,
+				VerticalOptions = LayoutOptions.Center
+			};
+			layout.Children.Add(label, Rectangle.FromLTRB(0, 0, 1, 1), AbsoluteLayoutFlags.All);
+			var tgr = new TapGestureRecognizer();
+			tgr.Tapped += (x, o) =>
+			{
+				Grid.SetColumnSpan(layout, Grid.GetColumnSpan(layout) + 1);
```

Comment:
This is just for my own edification; The UITest doesn't tap on it. Just forgot to remove it.